packages/ui-components/src/components/Map/TilePicker/index.js

```diff
@@ -5,3 +5,4 @@
  */
 
 export * from './TilePicker';
+export * from './constants';
```

Comment:
Was this needed in the end? I can't see it being used, but that might come in a later PR